### DISCUSSION TITLE
Windows App

### DISCUSSION THREAD

**Discussion by sguergachi** [notaapp/talk#289] [Feature Requests] [answered]
Just wanted to post here to request the Windows app. Caret was the best Windows Markdown editor around. Nota looks awesome. Cant wait to use it.

**Comment by astoilkov** [comment, marked as answer]
Hi Sammy,

Good to see you. Windows is part of the roadmap but it will probably take a while. We want to get the mac version right first.

Let the voting begin.

**Comment by sguergachi** [comment]
Thanks Antonio! Looking forward.
